react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -568,17 +919,24 @@ function _mapStateToProps(state, ownProps): Object {
 
     // Only the local participant won't have id for the time when the conference is not yet joined.
     const participant = participantID ? getParticipantById(state, participantID) : getLocalParticipant(state);
+    const { id } = participant;
     const isLocal = participant?.local ?? true;
+    const tracks = state['features/base/tracks'];
     const _videoTrack = isLocal
-        ? getLocalVideoTrack(state['features/base/tracks'])
-        : getTrackByMediaTypeAndParticipant(state['features/base/tracks'], MEDIA_TYPE.VIDEO, participantID);
+        ? getLocalVideoTrack(tracks) : getTrackByMediaTypeAndParticipant(tracks, MEDIA_TYPE.VIDEO, participantID);
     const _audioTrack = isLocal
-        ? getLocalAudioTrack(state['features/base/tracks'])
-        : getTrackByMediaTypeAndParticipant(state['features/base/tracks'], MEDIA_TYPE.AUDIO, participantID);
+        ? getLocalAudioTrack(tracks) : getTrackByMediaTypeAndParticipant(tracks, MEDIA_TYPE.AUDIO, participantID);
     const _currentLayout = getCurrentLayout(state);
     let size = {};
-    const { startSilent, disableProfile = false } = state['features/base/config'];
+    const {
+        startSilent,
+        disableLocalVideoFlip,
+        disableProfile = false,
```

Comment:
Why does this need a default? Its absence will be falsey too (undefined).

Reply:
It doesn't. will remove it.